src/patternfly/components/Dropdown/dropdown.scss

```diff
@@ -74,6 +78,7 @@ $pf-c-dropdown--breakpoint-map: build-breakpoint-map("base", "sm", "md", "lg", "
   --pf-c-dropdown__toggle--m-split-button--m-action--child--PaddingRight: var(--pf-global--spacer--sm);
   --pf-c-dropdown__toggle--m-split-button--m-action__toggle-button--MarginRight: calc(-1 * var(--pf-global--BorderWidth--sm));
   --pf-c-dropdown__toggle--m-split-button__toggle-check__input--TranslateY: #{pf-size-prem(-1px)};
+  --pf-c-dropdown__toggle--m-split-button__toggle-check__input--Visibility: auto;
```

Comment:
Should the value be `visible` instead since `auto` isn't a valid value for visibility? In the end it does get set to visible so may not be a huge deal.

Reply:
@thatblindgeye nice catch, thanks! Updated to `unset` since `visible` will set the checkbox to be visible if some parent element were set to `visiblity: hidden`. `unset` will allow `visibility` to continue to be inherited by default.